config/jobs/ci-infra/ci-infra-presubmits.yaml

```diff
@@ -27,7 +27,7 @@ presubmits:
         - --warnings=duplicate-job-refs
   - name: pull-ci-infra-prow-go-tests
     cluster: gardener-prow-build
-    run_if_changed: '^(prow\/*)'
+    run_if_changed: '^(prow\/*|Dockerfile|Makefile|go.(mod|sum))'
```

Comment:
`vendor` is missing.
Maybe it doesn't bring much value to restrict the test to specific paths. The overhead of always running it shouldn't be too high.

Reply:
I agree and removed the condition